TLM/TLM/Resources/whats_new.txt

```diff
@@ -1,3 +1,42 @@
+[Version] 11.6.5.3
+[Released] May 22nd 2022
+[Link] tmpe-v11653-test-22052022
+[Mod] Stable version released
+[New] Vietnamese Translation (DucAnhLam)
+[Updated] Translations update #1551 (aubergine18)
+[Fixed] Regression with version naming #1568 (krzychu124)
+[/Version]
+
+[Version] 11.6.5.3
+[Released] May 22nd 2022
+[Stable]
```

Comment:
This is going to be confusing to read - we've got 11.6.5.3 twice?

Also where did 11.6.5.2-hotfix-1 go (see PR https://github.com/CitiesSkylinesMods/TMPE/pull/1575)?

Suggestion: Leave the 11.6.5.2-hotfix-1 stuff as is, add a new 11.6.5.3 which is just hte translation stuff.

Reply:
Also, should we rename [11.6.5.2-hotfix-1 milestone](https://github.com/CitiesSkylinesMods/TMPE/milestone/66) to just be 11.6.5.3? If so is there other stuff that needs listing in the `whats_new.txt` and `CHANGELOG.md`?

Reply:
> This is going to be confusing to read - we've got 11.6.5.3 twice?

No idea how we handle Test changelog when Stable is released at the same time. You leave Stable changelog only?

For hotfix1 it was just one issue/PR theoretically we can clean up and merge, it was already released so no one will notice version change 😂

Reply:
eh,, nvm, I didn't notice `Closed` issues, but that doesn't matter, we can merge that